From a6a15b53dd0948fd0b1ef473d96f37724a70969b Mon Sep 17 00:00:00 2001
From: Evan Quan <evan.quan@amd.com>
Date: Thu, 20 Jul 2017 17:00:52 +0800
Subject: [PATCH 0811/4131] drm/amd/amdkcl: add dep_sync for amdgpu_cs.c -
 cooperate with "drm/amdgpu: add dep_sync for amdgpu job"

Change-Id: Iec6c5db4a48801e9ef2ca290fe99eb95622667c3
Signed-off-by: Evan Quan <evan.quan@amd.com>
Reviewed-by: Junwei Zhang <Jerry.Zhang@amd.com>
---
 drivers/gpu/drm/amd/amdgpu/amdgpu_cs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/amd/amdgpu/amdgpu_cs.c b/drivers/gpu/drm/amd/amdgpu/amdgpu_cs.c
index 91aac4a..96579b4 100644
--- a/drivers/gpu/drm/amd/amdgpu/amdgpu_cs.c
+++ b/drivers/gpu/drm/amd/amdgpu/amdgpu_cs.c
@@ -1120,7 +1120,7 @@ static int amdgpu_cs_dependencies(struct amdgpu_device *adev,
 		}
 	}
 
-	return amdgpu_sem_add_cs(p->ctx, p->job->ring, &p->job->sync);
+	return amdgpu_sem_add_cs(p->ctx, p->job->ring, &p->job->dep_sync);
 }
 
 static void amdgpu_cs_post_dependencies(struct amdgpu_cs_parser *p)
-- 
2.7.4

